### PR TITLE
Add sprint report 44

### PR DESCRIPTION
**Pull request by ChrisBr**
Co-authored-by: Björn Geuken <bgeuken@suse.de>
Co-authored-by: Dany Marcoux <dmarcoux@suse.com>
Co-authored-by: David Kang <dkang@suse.com>
Co-authored-by: Eduardo Navarro <enavarro@suse.com>
Co-authored-by: Henne Vogelsang <hvogel@opensuse.org>
Co-authored-by: Moisés Déniz Alemán <mdeniz@suse.com>
Co-authored-by: Saray Cabrera Padrón <scabrerapadron@suse.de>
Co-authored-by: Victor Pereira <vpereira@suse.com>

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *openbuildservice* ready!

Built with commit 1eb36654c8fd2e3df2d453a48cbd9c658d23edfc

https://deploy-preview-191--openbuildservice.netlify.com

File: _posts/development/2018-08-14-sprint-report-44.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+layout: post
+title: Highlights of the OBS frontend development - Sprint 44
+category: development
+author: "OBS Team"
+---
+
+People of the Builds! Another Sprint is over and here is what the OBS frontend team has achieved in the last two weeks (2018-07-30 to 2018-08-09). :smile:
+
+## Release of Azure Cloud Upload Feature
+Starting August was hot in Germany! And while talking about weather the cloud upload feature comes to our mind. :sunny: The OBS team released the Azure Cloud Upload, yes!. :cloud: Just have a look at the [blog post](https://openbuildservice.org/2018/08/02/cloud-upload-azure/) where it was announced and enjoy a step by step guid how to configure and use OBS to upload your images to the Azure Cloud. :sunglasses:
```
Comment (eduardoj) [inline _posts/development/2018-08-14-sprint-report-44.md]: s/guid/guide/

**Comment by dmarcoux** [comment]
I fixed my broken link, some typos/capitalization and sentences with wrong structures.

**Comment by ChrisBr** [comment]
thanks @dmarcoux

**Comment by ChrisBr** [comment]
> Nitpick: Don't forget to change the name of the file to be the same as the date when the blog entry will be published.

updated

> Nitpick: a reason of why imgur was introduced for images would be appreciated.

That was already done in the last sprint report. I find it neat!

**Pull request merged**